### ISSUE TITLE
How to download photos from more than one album?

### ISSUE THREAD

**Issue by Xm798** [icloud-photos-downloader/icloud_photos_downloader#438]
## Summary
As the title, I want to download photos from several albums, but it looks like only one album can be specified by the option `--album`.

## Context
I have many albums to store my photos, like photos taken by the Camera and retouched by other Apps. So I want to download all of them without putting them to an album.

**Comment by menkej** [comment]
This is correct. Can be a nice new feature and a good issue for a new developer to start in icloudpd.

**Comment by bersbersbers** [comment]
The requested feature might also include `--all-albums`.

**Comment by mmshivesh** [comment]
+1 and requesting ability to exclude specific albums from the list if needed

**Comment by AndreyNikiforov** [comment]
fixed in [v1.31.0](https://github.com/icloud-photos-downloader/icloud_photos_downloader/releases/tag/v1.31.0)
